app/factories/shopify/product_factory.rb

```diff
@@ -0,0 +1,59 @@
+module Shopify
+  class ProductFactory
+    def initialize(spree_product, shopify_product, logger = nil)
+      @logger = logger || default_logger
+      @spree_product = spree_product
+      @shopify_product = shopify_product
+    end
+
+    def perform
+      shopify_product.title = spree_product.name
+      shopify_product.body_html = surround_by_p_tags(spree_product.description)
+      shopify_product.created_at = spree_product.created_at
+      shopify_product.updated_at = spree_product.updated_at
+      shopify_product.published_at = spree_product.available_on
+      shopify_product.vendor = 'Glossier'
```

Comment:
I think we should avoid hard-coding integration-specific values into this extension. Where could this value come from instead?